README.md

```diff
@@ -5,6 +5,20 @@ Backend RESTful API for managing event data to be consumed by clients, like *web
 
 ## Project Setup
 
+Steps to setup a project based on Spring Boot:
```

Comment:
Maybe this should go in the wiki, since at the time a developer checks out this project, it will already be setup.  In addition, I would prefer not building from an IDE specifically
1. as different developers often have different preferences
1. being able to setup / build from the CLI allows builds to be portable, like for CI

Let's review this together some more in person.


That said, what would be useful in the README is how to setup the project more profoundly or specifically.  In this case, it may only be needing the equivalent of these Frontend [steps](https://github.com/ProvidenceGeeks/website-docs/wiki/Onboarding-Guide#frontend) but for the backend and documented [here](https://github.com/ProvidenceGeeks/website-docs/wiki/Onboarding-Guide#backend) such as
1.  How to install Java
1.  How to install Maven
1.  Confirm they are working

Essentially, at this moment from the README I would like to know how to run and build this project on my own machine, without being tied to a particular IDE.